Assets/GameAssets/Player/PlayerScripts/PlayerInteraction.cs

```diff
@@ -62,20 +73,29 @@ private void SetRaycast()
 
     /// <summary>
     /// checks if gameObject can be interacted with
-    ///     If so, calls GiveInteractableFeedback, which should give player feedback that the object is interactable
+    ///     If so, makes interactable text appear in HUD.  Otherwise disables text
     /// </summary>
     /// <param name="gameObject">GameObject that can potentially be interacted with</param>
     private void GiveInteractableFeedback(GameObject gameObject)
```

Comment:
could the name be more specific that gameObject? Because gameObject can also refer to the gameobject this script is attached to

Reply:
- The function already uses the name `interactable`.  
- The object is not necessarily interactable, that's why we're testing it.  That means it doesn't necessarily make sense to name it with anything that says `interactable`.
- Anything to the effect of `potentiallyInteractableGameObject` seems unnecessarily wordy
I'm not opposed to renaming it, but I don't have any better ideas

Reply:
Although I agree with your points, I still think we could use a different name than gameObject.